glusterd2/volgen/static.go

```diff
@@ -58,4 +58,23 @@ protocol/client`,
 cluster/replicate
 protocol/client`,
 	},
+	{
+		name: "gfproxy-fuse.graph",
+		content: `debug/io-stats
+performance/write-behind
+protocol/client`,
+	},
+	{
+		name: "gfproxy.graph",
```

Comment:
Shouldn't this be `gfproxyd.graph` as defined at `gfproxydTmpl` ?

Reply:
Done